monai/apps/deepgrow/transforms.py

```diff
@@ -0,0 +1,623 @@
+# Copyright 2020 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+A collection of "vanilla" transforms for spatial operations
+https://github.com/Project-MONAI/MONAI/wiki/MONAI_Design
+"""
+import json
+from typing import Optional, Sequence, Union
+
+import numpy as np
+
+from monai.config import KeysCollection
+from monai.transforms import Resize, SpatialCrop
+from monai.transforms.compose import MapTransform, Randomizable, Transform
+from monai.transforms.spatial.dictionary import InterpolateModeSequence
+from monai.transforms.utils import generate_spatial_bounding_box
+from monai.utils import InterpolateMode, ensure_tuple_rep, min_version, optional_import
+
+measure, _ = optional_import("skimage.measure", "0.14.2", min_version)
+distance_transform_cdt, _ = optional_import("scipy.ndimage.morphology", name="distance_transform_cdt")
+gaussian_filter, _ = optional_import("scipy.ndimage", name="gaussian_filter")
+
+
+class FindAllValidSlicesd(Transform):
```

Comment:
should inherit from MapTransform

Reply:
it's label specific.. i don't see any guidelines to mark it as maptransform vs transform when AddInitialSeedPointd is already dictionary based.